app/src/main/res/layout/fragment_countries_list.xml

```diff
@@ -6,49 +6,46 @@
     <androidx.constraintlayout.widget.ConstraintLayout
         android:layout_width="match_parent"
         android:layout_height="match_parent"
-        android:background="@drawable/bg1"
-        android:padding="10dp">
+        android:background="@color/colorPrimaryDark">
 
         <androidx.swiperefreshlayout.widget.SwipeRefreshLayout
             android:id="@+id/countries_reports_swipe_refresh"
             android:layout_width="0dp"
             android:layout_height="0dp"
-            android:layout_marginTop="8dp"
             app:layout_constraintBottom_toBottomOf="parent"
             app:layout_constraintEnd_toEndOf="parent"
             app:layout_constraintStart_toStartOf="parent"
-            app:layout_constraintTop_toBottomOf="@+id/countries_reports_search_view"
+            app:layout_constraintTop_toBottomOf="@+id/countries_search_view"
             app:refreshing="@{viewModel.isRefreshing}">
 
             <androidx.recyclerview.widget.RecyclerView
-                android:id="@+id/countries_reports"
                 android:layout_width="match_parent"
                 android:layout_height="0dp"
                 app:adapter="@{adapter}"
                 app:layoutManager="androidx.recyclerview.widget.LinearLayoutManager"
                 tools:listitem="@layout/item_country_report" />
-
         </androidx.swiperefreshlayout.widget.SwipeRefreshLayout>
 
         <SearchView
-            android:id="@+id/countries_reports_search_view"
+            android:id="@+id/countries_search_view"
             android:layout_width="0dp"
             android:layout_height="wrap_content"
-            android:background="@drawable/rounded_rect_bg"
+            android:background="@color/colorPrimary"
```

Comment:
I would vote for a different background. Maybe surface background (primaryDark?). And icon should be white IMO.

Reply:
done